.buildkite/scripts/build_pr.sh

```diff
@@ -77,6 +77,106 @@ if [[ "${GITHUB_PR_BASE_REPO}" != 'docs' ]]; then
+      git fetch origin "$GITHUB_PR_TARGET_BRANCH"
+      docs_diff=$(git diff --stat "origin/$GITHUB_PR_TARGET_BRANCH"...HEAD -- ./docs/guide)
+      ;;
+
+    "elasticsearch")
+      git fetch origin "$GITHUB_PR_TARGET_BRANCH"
+      docs_diff=$(git diff --stat "origin/$GITHUB_PR_TARGET_BRANCH"...HEAD -- ./buildSrc ./build-tools-internal ./build-tools/src/main/resources ./client ./docs ./modules/reindex/src/internalClusterTest/java/org/elasticsearch/client/documentation ./modules/reindex/src/test/java/org/elasticsearch/client/documentation ./plugins/examples ./server/src/internalClusterTest/java/org/elasticsearch/client/documentation ./server/src/main/resources/org/elasticsearch/common ./server/src/test/java/org/elasticsearch/client/documentation ./x-pack/docs ./x-pack/plugin/esql/qa/testFixtures/src/main/resources ./x-pack/plugin/sql/qa ./x-pack/qa/sql)
+      ;;
+
+    "elasticsearch-hadoop")
+      git fetch origin "$GITHUB_PR_TARGET_BRANCH"
+      docs_diff=$(git diff --stat "origin/$GITHUB_PR_TARGET_BRANCH"...HEAD -- ./docs/src/reference/asciidoc)
+      ;;
+
+    "elasticsearch-java")
+      git fetch origin "$GITHUB_PR_TARGET_BRANCH"
+      docs_diff=$(git diff --stat "origin/$GITHUB_PR_TARGET_BRANCH"...HEAD -- ./docs ./java-client/src/test/java/co/elastic/clients/documentation)
+      ;;
+
+    "elasticsearch-net")
+      git fetch origin "$GITHUB_PR_TARGET_BRANCH"
+      docs_diff=$(git diff --stat "origin/$GITHUB_PR_TARGET_BRANCH"...HEAD -- ./docs ./tests/Tests/Documentation)
+      ;;
+
+    "elasticsearch-py")
+      git fetch origin "$GITHUB_PR_TARGET_BRANCH"
+      docs_diff=$(git diff --stat "origin/$GITHUB_PR_TARGET_BRANCH"...HEAD -- ./docs/guide ./docs/examples)
+      ;;
+
+    "go-elasticsearch")
+      git fetch origin "$GITHUB_PR_TARGET_BRANCH"
+      docs_diff=$(git diff --stat "origin/$GITHUB_PR_TARGET_BRANCH"...HEAD -- ./.doc)
+      ;;
+
+    "enterprise-search-pubs")
+      git fetch origin "$GITHUB_PR_TARGET_BRANCH"
+      docs_diff=$(git diff --stat "origin/$GITHUB_PR_TARGET_BRANCH"...HEAD -- ./enterprise-search-docs ./workplace-search-docs ./app-search-docs ./esre-docs ./client-docs/app-search-javascript ./client-docs/app-search-node ./client-docs/workplace-search-node)
+      ;;
+
+    "enterprise-search-js")
+      git fetch origin "$GITHUB_PR_TARGET_BRANCH"
+      docs_diff=$(git diff --stat "origin/$GITHUB_PR_TARGET_BRANCH"...HEAD -- ./packages/enterprise-search/docs)
+      ;;
+
+    "esf" | "ingest-docs" | "observability-docs" | "stack-docs" | "x-pack-logstash")
+      git fetch origin "$GITHUB_PR_TARGET_BRANCH"
+      docs_diff=$(git diff --stat "origin/$GITHUB_PR_TARGET_BRANCH"...HEAD -- ./docs/en)
+      ;;
+
+    "kibana")
+      git fetch origin "$GITHUB_PR_TARGET_BRANCH"
+      docs_diff=$(git diff --stat "origin/$GITHUB_PR_TARGET_BRANCH"...HEAD -- ./docs ":(glob)x-pack/**/*.asciidoc" ":(glob)src/**/*.asciidoc" ":(glob)examples/**/*.asciidoc")
+      ;;
```

Comment:
@lcawl I might need your expertise on the link checking service to make sure we include the right file(s) here

Reply:
That's correct for every other repo. The Kibana repo, however, has a unique job that is defined outside of `conf.yaml`—link checking doc links from within the product.

Here's the function:

https://github.com/elastic/docs/blob/53a7366e0b92d07e821783b020f8cfbf5355e2ca/build_docs.pl#L345

There's a chance that it's as easy as adding these two files:

https://github.com/elastic/docs/blob/53a7366e0b92d07e821783b020f8cfbf5355e2ca/build_docs.pl#L393

https://github.com/elastic/docs/blob/53a7366e0b92d07e821783b020f8cfbf5355e2ca/build_docs.pl#L394

Reply:
Hey @kruskall. Thanks for chasing me down on this again. Yes, can you please remove the Kibana repo?

Reply:
Removed in ccc762d05bed7100331e4f396138c2127b8b7150